scripts/python_scripts/subquery_api.py

```diff
@@ -0,0 +1,127 @@
+from typing import List
+import requests
+import json
+
+
+class DeploymentInstance():
+
+    def __init__(self, **kwargs) -> None:
+
+        for key, value in kwargs.items():
+            setattr(self, key, value)
```

Comment:
It is better to explicitly specify which arguments class constructor accepts instead of settings everything from kwargs

Reply:
fixed, thanks